src/components/close-icon/index.tsx

```diff
@@ -0,0 +1,6 @@
+import { XMarkIcon } from '@heroicons/react/24/solid'
+import { FC } from 'react'
+
+export const CloseIcon: FC<React.ComponentProps<'svg'>> = ({ ...props }) => (
+  <XMarkIcon width={20} height={20} {...props} />
+)
```

Comment:
Make wrapper for icons with default width and height

Reply:
Not done -_-

Reply:
I think its a bad idia because we should use wrapper (div) around svg, will be easier pass props to svg (width and height) or used css